tests/tests/functional SFRA/Checkout/integration.js

```diff
@@ -166,5 +166,35 @@ registerSuite('Edq Cartridge Functional Test 2', {
 				})
 				.end()
 		},
+		"SFRA Check dropdown up/down keys - Global Intuitive": function() {
```

Comment:
I assume this test passes? You might also want to explicitly say here (in the comment or in the test name) what country is being tested.

Reply:
Yes, the test passed;the country being tested in this case is Canada, since the behavior replicated when I change country and default country is USA I've changed to Canada and tested.